site/styles/_forms.scss

```diff
@@ -37,11 +38,11 @@ s72-activatedevice-form,
     }
 
     .s72-form-group-password {
-      padding-right: $grid-gutter-width / 2;
+      padding-right: 2px;
```

Comment:
There are a few other uses of `$grid-gutter-width` - do you care about those?

Reply:
I removed `$grid-gutter-width` because the math result is 15px, which is too big.

We want the spacing of 2px only between input fields side by side:
![Screen Shot 2022-02-28 at 11 03 01 AM](https://user-images.githubusercontent.com/47130293/155901792-8bccdcc4-48e1-4d4d-8863-4c2001524769.png)

Reply:
Yep, I understand that 15px is too big for the design. My question was asking do you care about the other uses of `$grid-gutter-width` as well? Did you want to remove it elsewhere?

Reply:
Not at the moment, because other places which has this variable isn't related to input fields.